src/main/java/org/openrewrite/gradle/RewriteRecipeAuthorAttributionPlugin.java

```diff
@@ -56,7 +59,26 @@ public void apply(Project project) {
             copyAttribution.configure(task -> {
                 task.dependsOn(attr);
                 task.from(attr.get().getOutputDirectory());
-                task.into(new File(project.getBuildDir(), "resources/main/META-INF/rewrite/attribution"));
+                task.into(project.getLayout().getBuildDirectory().dir("resources/main/META-INF/rewrite/attribution").get().getAsFile());
+            });
+        }
+        
+        // Process YAML recipe files in resources
+        File yamlRecipeDir = new File(project.getProjectDir(), "src/main/resources/META-INF/rewrite");
+        if (yamlRecipeDir.isDirectory()) {
+            TaskProvider<RewriteRecipeAuthorAttributionTask> attr = tasks.register(
+                    "rewriteRecipeAuthorAttributionYaml", RewriteRecipeAuthorAttributionTask.class,
+                    task -> {
+                        task.setSources(yamlRecipeDir);
+                        task.setIsYamlRecipes(true);
+                    }
+            );
+
+            copyAttribution.configure(task -> {
+                task.dependsOn(attr);
+                task.from(attr.get().getOutputDirectory());
+                task.into(project.getLayout().getBuildDirectory().dir("resources/main/META-INF/rewrite/attribution").get().getAsFile());
+                task.setDuplicatesStrategy(DuplicatesStrategy.WARN);
```

Comment:
Not a blocking issue, but it's odd that we're seeing these duplicates; makes me think we're copying twice where we should only do so once.

Reply:
If this is benign we might as well move from `WARN` to ignoring the issue, but figured call it out for review first.

Reply:
Personally, I see it as indicative of a "duplicative touch." So I'd want to understand how we arranged the task to result in that effect and fix it correctly. Ignoring it would be akin to sweeping it under the rug and ignoring it. Only so much can fit under the rug though.

Reply:
Thanks! Rug lifted & swept out from underneath. Turns out `setAllSources` does what it says, so we were processing the resources twice. :)

Reply:
Gotcha. Is it possible to process resources on this same side? Reason that I ask is that all source also has the added consequence of capturing other languages for recipes. So if attribution is desirable for a customer using Kotlin the previous would have worked while the new no longer does.

Reply:
Sure! I'd initially held off on that as I didn't anticipate any `.kt` recipes that we'd want to document, but we can support it if we match resources by name, as opposed to asking Gradle which the Java / resources folders are. Thanks!

Reply:
Yeah, I don't think _we_ will likely use a language other than Java for recipe development, but a customer could choose to do so.